### DISCUSSION TITLE
Minhas leituras do primeiro semestre de 2024 – Beatriz Milz

### DISCUSSION THREAD

**Discussion by giscus** [beatrizmilz/blog-en#42] [General]
# Minhas leituras do primeiro semestre de 2024 – Beatriz Milz

https://beamilz.com/posts/2024-07-25-leituras-1sem/pt/

**Comment by garciadmp** [comment]
Muito legal a postagem! Além do conteúdo em si, ficou muito interessante o uso de dados para assuntos mais cotidianos. O goodreads, se não me engano, permite extrair em .csv a base de livros do nosso perfil

**Reply by beatrizmilz** [reply]
Muito obrigada! Eu não tinha usado o Goodreads, mas fiz uma conta depois do comentário. E realmente dá para exportar 🎉
